apps/ledger-live-desktop/src/renderer/screens/settings/WelcomeScreenSettings.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { RouterHistory, Match, Location } from "react-router-dom";
```

Comment:
I don't know if it's my local packages, but it seems those 3 types does not exist or are not exported from `react-router-dom`

Reply:
:feelthepainharold:

Reply:
thanks for the changes !